apptools/apptools-android-tests/tests.full.xml

```diff
@@ -68,6 +68,11 @@
           <test_script_entry>/opt/apptools-android-tests/apptools/mainactivity.py</test_script_entry>
         </description>
       </testcase>
+      <testcase component="Crosswalk App Tools/CLI" execution_type="auto" id="Crosswalk_cleanup_downloadfile" priority="P1" purpose="Android - Validate if clean up the unfinished file when downloads fail or are aborted" status="designed" type="Functional" subcase="2">
```

Comment:
Purpose: Validate if incomplete file is cleaned up when downing fails or is aborted.